app/forms/applicant_form.rb

```diff
@@ -3,32 +3,18 @@ class ApplicantForm
   include ActiveModel::Attributes
   include SessionPersistable
 
-  PERMANENT_ATTRIBUTES = %i[passporting over_60 employed].freeze
-  CONTINGENT_ATTRIBUTES = %i[partner_over_60 partner_employed].freeze
+  PROCEEDING_TYPES = { domestic_abuse: "DA001", other: "SE003" }.freeze
 
-  ATTRIBUTES = PERMANENT_ATTRIBUTES + CONTINGENT_ATTRIBUTES.freeze
+  PROCEEDING_ATTRIBUTE = %i[proceeding_type].freeze
+  PERMANENT_ATTRIBUTES = %i[passporting over_60 employed partner].freeze
 
-  attr_accessor :partner
+  ATTRIBUTES = PERMANENT_ATTRIBUTES + PROCEEDING_ATTRIBUTE.freeze
 
   PERMANENT_ATTRIBUTES.each do |attr|
     attribute attr, :boolean
     validates attr, inclusion: { in: [true, false] }
```

Comment:
I don't think we want this validation to be applied to the partner attribute if the Partner feature flag is switched off.